src/main/kotlin/racing/scene/PlayScene.kt

```diff
@@ -0,0 +1,61 @@
+package racing.scene
+
+import racing.engine.EnvironmentManager
+import racing.model.Car
+import racing.model.ScoreBoard
+import racing.port.OutputPainter
+import racing.port.RandomGeneratorPort
+import racing.utils.Constants
+
+class PlayScene(
+    private val outputPainter: OutputPainter,
+    private val scoreBoard: ScoreBoard,
+    private val randomGenerator: RandomGeneratorPort,
+    private val environmentManager: EnvironmentManager,
+) : Scene {
+
+    private var currentRound = 1
+    private var totalRound = 0
+    private var cars: List<Car> = emptyList()
```

Comment:
```suggestion
    private val cars: List<Car> = mutableListOf()
```
컬렉션은 컬렉션 내부에서 데이터를 add, remove 등 변경이 가능합니다.
컬렉션은 var가 아닌 val로 사용해주세요.

Reply:
var는 유지하지만 일급 객체로 변경하여 불변성을 지향해봤습니당 ㅎ